### PR TITLE
updates and style

### PR DESCRIPTION
**Pull request by privatezero**
Updates prior to externalizing DB doc info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,60 +28,69 @@ table of contents
...
 
-### installing mediamicroservices and dependencies###
+### installing mediamicroservices and dependencies
 once homebrew has been installed, you can install mediamicroservices. First, use the [tap](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/brew-tap.md) command. Then, you can install.
 
-Type brew tap mediamicroservices/mm into the command line.
+Type `brew tap mediamicroservices/mm` into the command line.
 
-Then, type brew install mediamicroservices/mm/mm into the command line. This command will install mediamicroservices to your computer.
+Then, type brew install mm into the command line. This command will install mediamicroservices to your computer.
```
Comment (dericed) [inline README.md]: tilde-quote commands

Comment (privatezero) [inline README.md]: Rats! I thought I caught all of those!

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ table of contents
     2. [installing mediamicroservices](https://github.com/mediamicroservices/mm#installing-mediamicroservices)
     3. [configuring mediamicroservices](https://github.com/mediamicroservices/mm#configuring-mediamicroservices)
         1. [variable explanations](https://github.com/mediamicroservices/mm#variable-explanations)
-3. [database configuration](#configuring-premisfixity-logging-database)
+3. [Database Instructions](#database-instructions)
```
Comment (retokromer) [inline README.md]: Why has this title caps and the others not? I suggest to unify either way.

Comment (privatezero) [inline README.md]: good point! I actually have been debating going through the doc and changing it to use capitalization across the board.

**Pull request merged**